.github/workflows/ingestion_client.yaml

```diff
@@ -0,0 +1,125 @@
+name: Build Ingestion Client Binaries
+
+on:
+  push:
+    branches: ['main', 'master']
+    paths: ['samples/ingestion/ingestion-client/**', '.github/workflows/ingestion_client.yaml']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
We are supposed to be using 1ES hosted pool, running the secure version of the public agents image.